i18n/locales/en_us.json

```diff
@@ -80,5 +83,5 @@
   "gifExportSettingSectionUploadDesc": "Upload as an animated GIF to a public URL.",
   "gifExportSettingSectionUncheckToPlay": "Uncheck to play the animation only one time.",
   "gifExportSettingSectionBuildingAnimatedGif": "Building animated GIF ...",
-  "gifExportSettingSectionYourImageAvailableAt": "Your image is now available at :"
+  "gifExportSettingSectionYourImageAvailableAt": "Your image is now available at"
```

Comment:
This is from a different PR, right?

Reply:
I applied this change to all the other branches 

Here I removed the colon since I was not sure if the string that will be translated should contain a colon. 

I removed it and included the colon here
this.i18n.gifExportSettingSectionYourImageAvailableAt() + ' :' + linkHtml;